Loop/Managers/DeviceDataManager.swift

```diff
@@ -576,7 +577,8 @@ final class DeviceDataManager {
                     }
 
                     let report = [
-                        Bundle.main.localizedNameAndVersion,
+                        "## Version",
+                        "* codeVersion: \(Bundle.main.localizedNameAndVersion)",
```

Comment:
This is the app name and version. not sure what "code version" is. That sounds like something that should be tracked by gitRevision/branch.

Reply:
The `## Version` header is a header for a section that is describing much more than just version. Could we call the section `## Build Details` ?

Reply:
I was about to suggest something like that.  Makes a lot more sense.

Reply:
Done